Nokia/pnx/res/values/config.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_supportDoubleTapWake">true</bool>
+    <bool name="config_dozePulsePickup">true</bool>
+	<bool name="config_automatic_brightness_available">true</bool>
+
+    <dimen name="status_bar_height">86px</dimen>
+    <dimen name="status_bar_height_portrait">86px</dimen>
+    <dimen name="status_bar_height_landscape">24dp</dimen>
+    <dimen name="rounded_corner_radius_top">110.0px</dimen>
+    <dimen name="rounded_corner_radius_bottom">110.0px</dimen>
+    <dimen name="notification_header_padding_top">10.0dip</dimen>
+    <dimen name="notification_header_padding_bottom">11.0dip</dimen>
+
+    <integer name="config_bluetooth_idle_cur_ma">1</integer>
+    <integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+    <integer name="config_bluetooth_rx_cur_ma">2</integer>
+    <integer name="config_bluetooth_tx_cur_ma">3</integer>
+    <integer name="config_screenBrightnessDark">1</integer>
+    <integer name="config_screenBrightnessDim">2</integer>
+    <integer name="config_screenBrightnessDoze">17</integer>
+    <integer name="config_screenBrightnessSettingDefault">128</integer>
+    <integer name="config_screenBrightnessSettingMaximum">255</integer>
+    <integer name="config_shutdownBatteryTemperature">600</integer>
+    <integer name="config_screenBrightnessSettingMinimum">1</integer>
+    <integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">8000</integer>
+	
+	<string name="config_icon_mask">M50 0C77.6 0 100 22.4 100 50C100 77.6 77.6 100 50 100C22.4 100 0 77.6 0 50C0 22.4 22.4 0 50 0Z</string>
```

Comment:
I have taken this from android.framework.base\res\values\strings.xml from stock, builds and works fine